content/docs/guide/gamemodes/sticky_jump.md

```diff
@@ -10,18 +10,16 @@ tags:
 
 ## What is Sticky Jump?
 
-Based on the Demoman class from Team Fortress 2, players shoot explosives that stick to surfaces and have control over their detonation which propels themselves throughout maps. Up to 3 of these arching bombs can be used at once to go at extremely high speeds, or they can be in alternation to pogo on the floor, the wall, or even in the air.
-
-Sticky Jump movement can be found in Team Fortress 2, however due to the high damage that stickybombs deal, it has extremely limited use in competitive games. Instead, most sticky jumping is done on special 'jump' maps and servers that give the player infinite health and ammo.
-
-The version of Sticky Jump implemented in Momentum Mod has a few key changes from Team Fortress 2. Most notably, all health and ammo is gone. Instead, players are limited to having 3 stickybombs out at any given time and further restrictions are done using special triggers in the maps.
+Based on the Demoman class from Team Fortress 2, players shoot explosives that stick to surfaces and have control over their detonation which propels themselves throughout maps. Up to 3 of these arcing bombs can be used at once to go at extremely high speeds, or they can be in alternation to pogo on the floor, the wall, or even in the air.
 
 ## History of Sticky Jump
 
-TODO
+Sticky Jump movement can be found in Team Fortress 2, but due to the high self-damage of stickybombs, its' prominence was limited within the main game. Most of sticky jumping is done on maps and servers specifically catered to it, inlcuding plugins that give unlimited ammo and set the players health to max on each tick, and allowed the use of some cheat-protected commands such as thirdperson.
+
+Momentum mods' implementation of Sticky Jump has a few key changes from Team Fortress 2. Ammo/reloading is removed, and instead of having to deal with health, players are limited to having 3 stickybombs out at any given time.
```

Comment:
Small grammar thing, it would be "Momentum Mod's implementation" due to Momentum Mod being a subject and not a collection of momentum mods

Reply:
Well, I'd done "Momentum mods'" because isn't the apostrophe after the s used for possession? Though to be fair it might be more consistent with other documentation, so I guess I'll change it.